src/test/recovery/t/139_archive_mode_always.pl

```diff
@@ -0,0 +1,82 @@
+# test wal persistance for archive_mode 'always'
+use strict;
+use warnings;
+use PostgresNode;
+use TestLib;
+use Test::More tests => 2;
+use File::Copy;
+
+# The idea of the test is to check that not archived WAL files are not removed
+# by creation of checkpoint (on master) or restartpoint (on standby).
+
+# Initialize master node, doing archives
```

Comment:
Mixing continious and non-continious forms of verbs sounds a bit awkward to me. I would suggest to re-phrase, for ex. "Initialize master node, doing archives" -> "Initialize master node and make archives", or smth similar

Reply:
1. Sorry, but I can't find any rule about it. If it's only a personal opinion, then I will ignore it, as none of us is native. If it's a rule, then please give a link.
2. The original comment was left by Alvaro Herrera in his [patch](https://github.com/postgres/postgres/commit/49148645f7f30f461b09618b5342b37425f33b22#diff-b57990e9ef44c06b531f9dd2279d95e1a344348d3298f81820a4eb4690fcb205R9) to PG. He, the author and other reviewers are probably not natives too, but all of them are OK with this.
3. (Deep) IMO, but we shouldn't care SO much about such things here or in any other patch (doesn't matter whether we made this changes or not).